### PR TITLE
Feature/tester permissions

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
### Description of the Change
Add permissions to tester account.

### CODE REVIEW

File: deploy/iroha/genesis.block [hunk 1]
```diff
@@ -137,30 +137,36 @@
...
-                    "can_create_account",
                     "can_get_all_txs",
                     "can_get_domain_acc_detail",
-                    "can_read_assets",
-                    "can_grant_can_set_my_quorum",
-                    "can_grant_can_add_my_signatory",
-                    "can_grant_can_transfer_my_assets",
-                    "can_get_my_signatories",
-                    "can_remove_signatory"
+                    "can_read_assets"
```
Comment (Mingela) [inline deploy/iroha/genesis.block]: line 162 has the same

File: deploy/iroha/genesis.block [hunk 2]
```diff
@@ -137,30 +137,36 @@
...
-                    "can_set_quorum",
-                    "can_add_signatory",
-                    "can_create_account",
                     "can_get_all_txs",
                     "can_get_domain_acc_detail",
-                    "can_read_assets",
-                    "can_grant_can_set_my_quorum",
-                    "can_grant_can_add_my_signatory",
-                    "can_grant_can_transfer_my_assets",
-                    "can_get_my_signatories",
```
Comment (Mingela) [inline deploy/iroha/genesis.block]: it's deleted and not added anywhere else

Comment (Alexey-N-Chernyshov) [reply deploy/iroha/genesis.block]: there is can_get_all_signatories

File: deploy/iroha/genesis.block (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply deploy/iroha/genesis.block]: fixed

**Pull request merged**